crest/Assets/Crest/Crest/Shaders/OceanEmission.hlsl

```diff
@@ -70,7 +70,8 @@ void ApplyCaustics
 {
 	// could sample from the screen space shadow texture to attenuate this..
 	// underwater caustics - dedicated to P
-	const float3 scenePosUV = WorldToUV(i_scenePos.xz, cascadeData1, _LD_SliceIndex + 1);
+	// TODO: Investigate if necessary anymore.
```

Comment:
soft vote for looking into this if its fast rather than adding a todo, but also fine with it being deferred.

Reply:
I've removed it. Wasn't necessary anymore after testing.